### ISSUE TITLE
Need to add Quality Assurance target type to Organization

### ISSUE THREAD

**Issue by CTITechPlanning** [CredentialTransparencyInitiative/vocabularies#3]
From https://docs.google.com/document/d/13NxPR4l4iAuZ5IseRtSzfh61Jj3F6q15NMP2ToYjDhM/edit

The example data:
This QA body accredits, endorses, approves, or recognizes:
- Educational certificates
- AA degrees
- Industry educational certificates outside of higher education

**Comment by CTITechPlanning** [comment]
Following discussions, the current approach is to replace serviceCategory with qaTargetType (or similarly named property) with a vocabulary of Credential, Organization, Assessment, and Learning Opportunity

**Comment by stuartasutton** [comment]
Following the naming convention noted in my comment on issue #9, I propose that the property be named "qualityAssuranceTargetType" (a bit wordy, but OK) defined as "The type of resource that the agent accredits, endorses, approves, or recognizes."

And that the related vocabulary class be named QualityAssuredResource" with the following concepts defined: "Credential", "CredentialOrganization", "CredentialPerson", "Assessment", and "Learning Opportunity".

**Comment by stuartasutton** [comment]
Note that the vocabulary concepts "Credential", "CredentialOrganization", and "CredentialPerson" are problematic with schema.org because we already have these notions defined as classes of things.

**Comment by stuartasutton** [comment]
Consider "qualityAssuranceTargetType" referencing the existing classes "Credential", "CredentialOrganization", and "CredentialPerson". Create classes for for "Assessment" and "LeaningOpportunity" so we can handle them in the same way. 

Note that having classes for "Assessment" and "LeaningOpportunity" would not be a bad idea anyway!

**Comment by stuartasutton** [comment]
I created the the qualityAssuranceTargetType property, [1] as well as the LearningOpportunity class [2] and the Assessment class [3]. That now supports ranges of all four classes: "Credential", "CredentialOrganization", and "CredentialPerson", "Assessment" and "LeaningOpportunity".

[1] http://www.credreg.net/page/propertiesviewer#qualityAssuranceTargetType
[2] http://www.credreg.net/page/propertiesviewer#LearningOpportunity
[3] http://www.credreg.net/page/propertiesviewer#Assessment
